Adafruit_Sensor.h

```diff
@@ -136,8 +138,11 @@ typedef struct {
     float current;           /**< current in milliamps (mA) */
     float voltage;           /**< voltage in volts (V) */
     float tvoc;              /**< Total Volatile Organic Compounds, in ppb */
-    sensors_color_t color;   /**< color in RGB component values */
-  };                         ///< Union for the wide ranges of data we can carry
+    int32_t voc_index; /**< VOC (Volatile Organic Compound) index where 100 is
+                          normal */
+    int32_t nox_index; /**< NOx (Nitrogen Oxides) index where 100 is normal */
```

Comment:
Could we keep this as a `float`?

Reply:
Also - if this is unit-less, please add that to the comment(s) and documentation